speeduino/corrections.ino

```diff
@@ -401,19 +389,19 @@ uint16_t correctionAccel()
     
       //Check for deceleration (Deceleration adjustment not yet supported)
       //Also check for only very small movement (Movement less than or equal to 2% is ignored). This not only means we can skip the lookup, but helps reduce false triggering around 0-2% throttle openings
-      if (TPS_change <= 2)
+      if (configPage2.taeThresh <= 8)
```

Comment:
Yeah it was that TPS movement is no more calculated in the corrections.ino . It must use only tpsDOT from the sensors.ino. I think too it probably is better remove from there, and apply 1% TPS movement filter at the tpsDOT calculation.

Reply:
Fixed in the [cf8ed56](https://github.com/noisymime/speeduino/pull/784/commits/cf8ed56be257b4152e14ccfc88f2c1888e8fe302)
Setting the `configPage2.taeThresh <= 1` should mathematically and functionally result in the needed 1% TPS movement treshold, so that it do not break any setups.